src/amo/pages/SearchPage/index.js

```diff
@@ -43,17 +42,9 @@ export class SearchPageBase extends React.Component<InternalProps> {
     let shouldRedirect = false;
     const newFilters = { ...filters };
 
-    // The legacy frontend uses `all` to express "all platforms" but we now use
-    // no parameter (empty) to target all platforms. In addition, the query
-    // parameter `platform` is mapped to the `operatingSystem` filter.
-    // See: https://github.com/mozilla/addons-frontend/issues/3870.
-    if (
-      newFilters.operatingSystem &&
-      newFilters.operatingSystem.toLowerCase() === 'all'
-    ) {
-      log.info('`operatingSystem` filter is set to "all", omitting it.');
-      delete newFilters.operatingSystem;
-
+    // We removed the `platform` parameter, so if it's present, remove it and
```

Comment:
```suggestion
    // We removed the `platform` parameter, so if it's present, ignore it and
```

(it's a nit but I believe we don't remove anything, we simply ignore it and don't add anything special to `newFilters`)